lib/core/src/lib/card-view/components/card-view-keyvaluepairsitem/card-view-keyvaluepairsitem.component.ts

```diff
@@ -22,10 +22,11 @@ import { MatTableDataSource } from '@angular/material/table';
 import { BaseCardView } from '../base-card-view';
 
 @Component({
-    selector: 'adf-card-view-keyvaluepairsitem',
+    selector: 'adf-card-view-key-value-pairs-item',
```

Comment:
Should we maybe rename the files for this component as well? To keep consistency